translations/harbour-berlin-vegan-de.ts

```diff
@@ -210,5 +210,9 @@
         <source>Search...</source>
         <translation>Suche...</translation>
     </message>
+    <message id="id-no-entries-available">
+        <source>No entries available</source>
+        <translation>Keine Einträge vorhanden</translation>
```

Comment:
Hmm… I think these strings might confuse the user B/C it could sound like "No entries available" … in the database.

Maybe we could go for sth. like "No search or filter results"/"Keine Filter-/Suchergebnisse" or shorter "No matches"/"Keine Treffer"?

If you use sth. longer, Label with ```wrapMode: Text.WordWrap``` could help you.